src/functions.php

```diff
@@ -43,11 +43,12 @@ function captureException(\Throwable $exception): ?EventId
 /**
  * Captures a new event using the provided data.
  *
- * @param array<string, mixed> $payload The data of the event being captured
+ * @param Event                  $event The event being captured
+ * @param \Sentry\EventHint|null $hint  May contain additional information about the original exception
```

Comment:
Since the class is already imported you can just reference it by its name rather than FQCN